<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NetGameWindow</class>
 <widget class="QMainWindow" name="NetGameWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>660</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>660</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>660</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Gobang</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>20</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_2">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>50</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>20</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>LocalPort:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>50</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>GuestPort:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>654</x>
      <y>80</y>
      <width>61</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>OK</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>80</y>
      <width>61</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Cancel</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>654</x>
      <y>140</y>
      <width>127</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Exit</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>654</x>
      <y>110</y>
      <width>127</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Choose Role</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>NetGameWindow</receiver>
   <slot>ipPortSetting()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>714</x>
     <y>102</y>
    </hint>
    <hint type="destinationlabel">
     <x>736</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>NetGameWindow</receiver>
   <slot>CancelConnect()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>735</x>
     <y>98</y>
    </hint>
    <hint type="destinationlabel">
     <x>761</x>
     <y>137</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_3</sender>
   <signal>clicked()</signal>
   <receiver>NetGameWindow</receiver>
   <slot>GameExit()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>694</x>
     <y>184</y>
    </hint>
    <hint type="destinationlabel">
     <x>677</x>
     <y>180</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_4</sender>
   <signal>clicked()</signal>
   <receiver>NetGameWindow</receiver>
   <slot>ChooseRole()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>661</x>
     <y>153</y>
    </hint>
    <hint type="destinationlabel">
     <x>693</x>
     <y>260</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>ipPortSetting()</slot>
  <slot>CancelConnect()</slot>
  <slot>GameExit()</slot>
  <slot>ChooseRole()</slot>
 </slots>
</ui>
